packages/instrumentation-amqplib/package.json

```diff
@@ -0,0 +1,68 @@
+{
+    "name": "opentelemetry-instrumentation-amqplib",
+    "version": "0.2.2",
+    "description": "open telemetry instrumentation for the `amqplib` package",
+    "keywords": [
+        "amqplib",
+        "opentelemetry",
+        "rabbitmq",
+        "AMQP 0-9-1"
+    ],
+    "homepage": "https://github.com/aspecto-io/opentelemetry-amqplib",
+    "license": "Apache-2.0",
+    "main": "dist/src/index.js",
+    "files": [
+        "dist/src/**/*.js",
+        "dist/src/**/*.d.ts",
+        "LICENSE",
+        "README.md"
+    ],
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/aspecto-io/opentelemetry-amqplib.git"
+    },
+    "scripts": {
+        "build": "tsc",
+        "prepare": "yarn run build",
+        "test": "mocha",
+        "watch": "tsc -w",
+        "version:update": "node ../../scripts/version-update.js",
+        "version": "yarn run version:update",
+        "test-all-versions": "tav",
```

Comment:
can call `tav` directly from `test:ci`

Reply:
Yeah you a right

I found it more descriptive to call `test-all-versions` while developing, so I can trigger a call locally without calling `someting:ci`.
If you think it is better in the compact style, I am open to change that

Reply:
imo it's redundant - too many scripts.
Also doesn't align with the rest of the repo.
But as you choose.